README.md

```diff
@@ -513,6 +513,10 @@ Undefined steps snippets are printed in javascript by default.
 Custom snippet syntaxes can be used with `--snippet-syntax <FILE>`.
 See [here](/features/step_definition_snippets_custom_syntax.feature) for an example.
 
+### Profiles
+
+In order to store and reuse commonly used CLI options, you can add a `cucumber.js` file to your project root directory. The file should export an object where the key is the profile name and the value is a string of CLI options. The profile can be applied with `-p <NAME>` or `--profile <NAME>`. This was prepend the profile's CLI options to the execution. The options are prepended so they can be overriden by supplied options. Multiple profiles can be specified at a time. If no profile is specified and a profile named `default` exists, it will be applied.
```

Comment:
Weird grammatical structure: "This was prepend the profile's CLI options to the execution"